File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/issuedTicket/exception/IssuedTicketUserNotMatchedException.java
package band.gosrock.domain.domains.issuedTicket.exception;


import band.gosrock.common.exception.DuDoongCodeException;
import band.gosrock.common.exception.GlobalErrorCode;

public class IssuedTicketUserNotMatchedException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new IssuedTicketUserNotMatchedException();

    private IssuedTicketUserNotMatchedException() {
        super(GlobalErrorCode.ISSUED_TICKET_NOT_MATCHED_USER);
    }
}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/issuedTicket/exception/IssuedTicketUserNotMatchedException.kt
package band.gosrock.domain.domains.issuedTicket.exception

import band.gosrock.common.exception.DuDoongCodeException

class IssuedTicketUserNotMatchedException private constructor() : DuDoongCodeException(IssuedTicketErrorCode.ISSUED_TICKET_NOT_MATCHED_USER) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = IssuedTicketUserNotMatchedException()
    }
}
